src/api/search.api.js

```diff
@@ -258,6 +258,7 @@ async function searchLayerFeatures(outputProjection, queryString, layer, lang, c
                     ...locationContent,
                     resultType: SearchResultTypes.FEATURE,
                     title,
+                    layer,
```

Comment:
don't forget to document the JSDoc for the `LayerFeatureSearchResult` typedef with this new addition 😉

Reply:
ah yes, of course!